bindings/python/triplestore/units.py

```diff
@@ -0,0 +1,250 @@
+"""Populates a pint unit registry from an ontology.
+
+Creates a Generator for the lines in the Pint unit registry.
+
+"""
+from email.policy import default
+from pint import UnitRegistry, Quantity
+import re
+from triplestore import Triplestore, RDFS
+import warnings
+
+def load_qudt():
+    print("Loading QUDT unit ontology.")
```

Comment:
Avoid print statements in libraries. Use the logging module when needed.

In this case it is fine to just remove the print statement